docs/content/production.html.md

```diff
@@ -347,6 +347,19 @@ The `controller`, `router`, and `blobstore` components store data in
 a PostgreSQL cluster managed by Flynn. Their databases may be accessed by
 running `flynn -a $APP_NAME pg psql`.
 
+### Recovery
+
+In the event that a cluster is having difficulties with apps, either internal
+or external, not starting after a cluster reboots, it may be necessary to run 
+the automated cluster repair tool. 
+
+Before attempting recovery, [perform a backup](#backup/restore) and [collect the logs](#retrieving-logs).
+Send the logs to [logs@flynn.io](mailto:logs@flynn.io) to debug the cluster
+and help determine what caused it to fail.
+
+Next, run `flynn-host fix` from any node in the cluster to start an automated
```

Comment:
We should advise users to specify `--min-hosts=N` (the minimum expected number of hosts)